frontend/src/Pages/ResourcesManagement.tsx

```diff
@@ -284,11 +281,10 @@ export default function ResourcesManagement() {
     }
 
     return (
-        <AuthenticatedLayout title="Categories">
-            <PageNav
-                user={auth.user}
-                path={['Settings', 'Resource Management']}
-            />
+        <AuthenticatedLayout
+            title="Categories"
+            path={['Settings', 'Resource Management']}
```

Comment:
Same with this one? Or are we leaving this to be changed in the fix for #345 ?

Reply:
I will remove it for now since there's no "Settings" page. We can have [#345](https://github.com/UnlockedLabs/UnlockEdv2/issues/345) be more focused on whether we have a Breadcrumb UI or not.